### ISSUE TITLE
Show seed phrase should include QR

### ISSUE THREAD

**Issue by bdresser** [MetaMask/Design#142]
In addition to showing the pneumonic, we should also display a QR to let users easily import wallets elsewhere.

Came up in a conversation on mobile, also relevant to the extension.

**Comment by brunobar79** [comment]
@bdresser  For the record, we only need the design for the import seed phrase flow, which I assume it could be a QR code icon on the bottom right of the seed phrase text area.
I'm moving fwd with this approach for now but I'm happy to tweak the design once this is ready.

All the other cases are already covered (Import / export private key & export seed phrase)

**Comment by brunobar79** [comment]
Demo of what I've implemented so far: http://recordit.co/dXpW13Rauf
